CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ CHANGELOG
 - Add support for custom actions on login nodes.
 
 **BUG FIXES**
+
+- Fix cluster deletion failure when enable placement group.
```

Comment:
minor: "when placement group is enabled"

Reply:
Done!